modules/minimint-wallet/src/lib.rs

```diff
@@ -343,6 +343,7 @@ impl FederationModule for Wallet {
 
     async fn end_consensus_epoch<'a>(
         &'a self,
+        _consensus_peers: &HashSet<PeerId>,
```

Comment:
Maybe add a `FIXME` comment/open an issue since the problem exists for PSBTs as well?

Reply:
For the LN module as well once we have the decryption stuff from @justinmoon's branch?